pages/index.tsx

```diff
@@ -1,6 +1,7 @@
 import Contact from "@/components/general/Contact";
 import Values from "@/components/general/OurValues";
 import Head from "next/head";
+import FeaturedWorkSection from "../components/general/FeaturedWorkSection";
```

Comment:
The name of the file and component should be in the same format as the one team has already done.

Reply:
I change the name from FeaturedWorkSection to FeaturedWork